code/controllers/subsystems/machines.dm

```diff
@@ -19,18 +12,17 @@ SUBSYSTEM_DEF(machines)
 
 	var/current_step = SSMACHINES_PIPENETS
 
-	var/cost_pipenets      = 0
-	var/cost_machinery     = 0
-	var/cost_powernets     = 0
-	var/cost_power_objects = 0
+	var/cost_pipenets		= 0
+	var/cost_machinery		= 0
+	var/cost_powernets		= 0
+	var/cost_power_objects	= 0
```

Comment:
Style nit: Can we use spaces for vertical alignment of stuff at the end of the line because tabs will render differently between different systems like this.

Reply:
Fixed!